pennylane/ops/qutrit/state_preparation.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This submodule contains the discrete-variable quantum operations concerned
+with preparing a certain state on the qutrit device.
+"""
+# pylint:disable=abstract-method,arguments-differ,protected-access,no-member
+from pennylane import numpy as np
+from pennylane import math
+from pennylane.operation import AnyWires, StatePrep
+from pennylane.templates.state_preparations import QutritBasisStatePreparation
+from pennylane.wires import Wires, WireError
+
+state_prep_ops = {"QutritBasisState"}
+
+
+class QutritBasisState(StatePrep):
```

Comment:
Please add the `QutritBasisState` operator to `doc/introduction/operations.rst` in the appropriate section, see `BasisState` for reference

Reply:
Done!